spoon-runner/src/main/java/com/squareup/spoon/html/HtmlRenderer.java

```diff
@@ -74,21 +75,17 @@ private void generateCssFromLess() {
       String less = Resources.toString(getClass().getResource("/spoon.less"), UTF_8);
       String css = compiler.compile(less);
       File cssFile = FileUtils.getFile(output, STATIC_DIRECTORY, "spoon.css");
-      FileUtils.writeStringToFile(cssFile, css);
+      FileUtils.writeStringToFile(cssFile, css, UTF_8);
     } catch (Exception e) {
       throw new RuntimeException(e);
     }
   }
 
   private void writeResultJson() {
-    FileWriter result = null;
-    try {
-      result = new FileWriter(new File(output, "result.json"));
+    try (Writer result = new PrintWriter(new File(output, "result.json"), "UTF-8")) {
```

Comment:
No, constructor with Charset is private, but I can use this with Charset instead

`new BufferedWriter(new OutputStreamWriter(new FileOutputStream(new File(output, "result.json")), UTF_8)`

It is how it's constructs within PrintWriter